packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -42,7 +42,13 @@ enum _PlatformViewState {
 /// Android [View](https://developer.android.com/reference/android/view/View).
 ///
 /// The render object's layout behavior is to fill all available space, the parent of this object must
-/// provide bounded layout constraints
+/// provide bounded layout constraints.
+///
+/// RenderAndroidView participates in Flutter's [GestureArena]s, and dispatches touch events to the
+/// Android view iff it won the arena. Specific gestures that should be dispatched to the Android
+/// view can be specified in [RenderAndroidView.gestureRecognizers]. If
+/// [RenderAndroidView.gestureRecognizers] is empty, the gesture will be dispatched to the Android
+/// view iff it was not claimed by any other gesture recognizer.
 ///
```

Comment:
Wouldn't we want a mode where nothing is routed to the Android view for showing static content?

Reply:
This can be achieved by setting a transparent hit test behavior